src/components/WeeklyForecast/WeeklyForecast.js

```diff
@@ -18,7 +20,7 @@ class WeeklyForecast extends React.Component {
     const weatherDays = this.props.forecast.map (day =>{
       const dayKey = day.date
       const findSelectedDay = this.props.selectedDays.findIndex(activeDay => activeDay.date === dayKey)
-      const findRecommendedDay = this.props.recommendedDays.findIndex(recDay => recDay.date == dayKey)
+      const findRecommendedDay = this.props.recommendedDays.findIndex(recDay => recDay.date === dayKey)
       
       return (
         <div className={`day-container ${(findSelectedDay === -1) ? '' : 'active'} ${(findRecommendedDay === -1) ? '' : 'recommended'}`} key={day.date} id={day.date} tabIndex={0} onClick={(event) => this.props.selectDay(event.currentTarget.id)}>
```

Comment:
I thiiiiink (like 73% sure) these ternaries can be simplified to `variable === value && "class"` instead of using an empty string in the ternary and it'll still work.